package.json

```diff
@@ -2788,15 +2788,15 @@
         "@docker/sdk": "^1.0.3",
         "@grpc/grpc-js": "^1.2.2",
         "dayjs": "^1.10.2",
-        "dockerfile-language-server-nodejs": "^0.2.1",
+        "dockerfile-language-server-nodejs": "^0.2.2",
         "dockerode": "^3.2.1",
         "fs-extra": "^9.0.1",
         "glob": "^7.1.6",
         "gradle-to-js": "^2.0.0",
         "handlebars": "^4.7.6",
         "request": "^2.88.2",
         "request-promise-native": "^1.0.9",
-        "semver": "7.3.2",
+        "semver": "^7.3.4",
```

Comment:
I've already tested with success, but need to make absolutely certain this doesn't regress and cause #2575 to come back.